src/components/Label/Label.tsx

```diff
@@ -1,42 +1,22 @@
 import { JSX } from 'react';
-import { Group, InputLabel, Text, Tooltip } from '@mantine/core';
+import { Group, InputLabel, Text } from '@mantine/core';
 
-import CopyUriButton from '@/components/CopyUriButton/CopyUriButton';
 import { InfoBox } from '@/components/InfoBox/InfoBox';
-import { Uri } from '@/types/types';
 
 interface Props {
-  name: string;
+  name: string | JSX.Element;
```

Comment:
```suggestion
  name: JSX.Element;
```
String is part of JSX.Element